CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 # Copyright 2013-2020 the openage authors. See copying.md for legal info.
 
-cmake_minimum_required(VERSION 3.12)
+cmake_minimum_required(VERSION 3.16)
```

Comment:
can you annotate why we need that version now ("finding numpy with the findpython3 module" or something)